test/Microsoft.ML.Predictor.Tests/TestPredictors.cs

```diff
@@ -781,7 +781,7 @@ public void TestMultiClassEnsembleCombiner()
 
             var predictors = new PredictorModel[]
             {
-                LightGbm.TrainMultiClass(Env, new LightGbmArguments
+                LightGbm.TrainMultiClass(Env, new Options
```

Comment:
Do you have tests for the catalog entries? I don't see them here &mdash; they didn't show up as a change, so it looks like they aren't being tested.

Reply:
good catch. I have updated several tests so they exercise the public API catalog extensions 

---
In reply to: [251537729](https://github.com/dotnet/machinelearning/pull/2261#discussion_r251537729) [](ancestors = 251537729)